integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/datamap/TestDataMapCommand.scala

```diff
@@ -81,8 +82,25 @@ class TestDataMapCommand extends QueryTest with BeforeAndAfterAll {
     assert(dataMapSchemaList.size() == 3)
   }
 
+  test("test show datamap without preaggregate") {
+    sql("drop table if exists datamapshowtest")
+    sql("create table datamapshowtest (a string, b string, c string) stored by 'carbondata'")
+    sql("create datamap datamap1 on table datamapshowtest using 'new.class' dmproperties('key'='value')")
+    sql("create datamap datamap2 on table datamapshowtest using 'new.class' dmproperties('key'='value')")
+    checkExistence(sql("show datamap on table datamapshowtest"), true, "datamap1", "datamap2", "-NA-", "new.class")
```

Comment:
In output string, it should show `datamapshowtest` instead of `-NA-` for the table name, right?

Reply:
No, it is NA only. The associated table available only for preaggregate datamaps